src/fastfilter.h

```diff
@@ -7,10 +7,121 @@
 
 #include "random.h"
 #include "serialize.h"
+#include "util.h"
+#include <cmath>
 #include <vector>
 
+#define LN2SQUARED 0.4804530139182014246671025263266649717305529515945455
+#define LN2 0.6931471805599453094172321214581765680755001343602552
+#define MIN_N_HASH_FUNC 1
+#define MAX_N_HASH_FUNC 32
+const uint32_t MAX_32_BIT = (int)pow(2, 32) - 1;
 class uint256;
```

Comment:
are you intending MAX_32_BIT to be a max uint32 or an int32?   Looks like the calculation is for an int32.  Better to use std::numeric_limits here also to be more clear.

Reply:
Good suggestion. Done.